tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -123,7 +123,7 @@ class VerifiableConsumer(BackgroundThreadService):
     logs = {
         "verifiable_consumer_stdout": {
             "path": STDOUT_CAPTURE,
-            "collect_default": False},
```

Comment:
Verifiable consumer necessarily prints something for every record right? I would think that could easily grow larger than the debug log. But in any case it doesn't look like anything is collected by default for VerifiableConsumer, so I'm not sure why any other log file size would be relevant for comparison...